content/espanol/sql_workshop/activity7/_index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Actividad 7: Usa el comando MIN"
+description: "Usa el comando MIN"
+date: 2020-07-10
+prereq: "Comando SELECT, comando WHERE, comando MAX/MIN"
+difficulty: "Principiante"
+draft: false
+---
+<!-- Links for javascript and CSS needed for drop down logic -->
+<link rel="stylesheet" href="../default/_default.css" type="text/css"></link>
+<link rel="stylesheet" href="../default/_type.css" type="text/css"></link>
+<link rel="stylesheet" href="_activity7.css" type="text/css"></link>
+
+<script type="text/javascript" src="../default/_default.js"></script>
+<script type="text/javascript" src="../default/_type.js"></script>
+<script type="text/javascript" src="../default/alasql.js"></script>
+<script type="text/javascript" src="../default/db.js"></script>
+<script type="text/javascript" src="_activity7.js"></script>
+
+## Tarea 7: Encuentra la llave para conseguir el Cañón de Confeti.
+
+Encontraste la ubicación del Legendario Cañon de Confeti pero está encerrado detrás de una puerta de titanio y solo puede ser abierta con la llave más pequeña del planeta.
+
+**Para ayudarte, la Federación Gálactica te ha enviado un recordatorio de la actividad anterior sobre cómo usar el comando `MIN()`.**
+
+![Explain](../activity5/assets/max_min.png)
+
+#### Usando la base de datos llamada <i>'items'</i> (artículos) con columnas etiquetadas como <i>'object'</i> (objeto) y <i>'height'</i>(altura), encuentra la <i>'key'</i> (llave) con la menor altura. 
+{{% notice tip %}}
+Puedes mostrar la base de datos entera usando el comando de la actividad 1.
+
+* Pista 1: Recuerda el consejo de la Federación Gálactica: `SELECT [columna/propiedad de columna] FROM [base de datos] [condicionales opcionales];`
+
+* Pista 2: Condicionales `WHERE [column_name] [IN()/NOT IN()];` 
+
+* Pista 3: ¿Necesitar usar `IN()` o `NOT IN()` para mostrar la 'key' (llave)?
+{{% /notice%}}
+<!-- SQL Type In Activity -->
+
+{{< rawhtml >}}
+
+  <div class="content_scaler">
+    <div class="terminal_div" id="terminal_div">
+      <div class = "outer">
+        <h3 id = "commands" contenteditable="true" onclick="placeholder()">Escribe el comando aquí</h3>
+      </div>
+      <div class = "prev">
+        <h3 id = "prev"></h3>
+      </div>
+      <div style="clear: both;"></div> 
+      <button class="button button1" onclick="sql()"> Enter </button>
```

Comment:
Translate

Reply:
We won't translate Enter in any of the instances as is commonly used in spanish too.